api/src/main/java/org/openmrs/util/UpgradeUtil.java

```diff
@@ -38,19 +39,17 @@ public static Integer getConceptIdForUnits(String units) {
 		String appDataDir = OpenmrsUtil.getApplicationDataDirectory();
 		Properties props = new Properties();
 		String conceptId = null;
-		String filePath = new StringBuilder(appDataDir)
-				.append(System.getProperty("file.separator"))
-				.append(DatabaseUtil.ORDER_ENTRY_UPGRADE_SETTINGS_FILENAME).toString();
-
-		try (FileInputStream fis = new FileInputStream(filePath)) {
-
+		try {
+			FileInputStream fis = new FileInputStream(
```

Comment:
@MalshaL please also use the try-with-resource statement
http://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
instead of calling the close() yourself.

this has just been fixed in https://issues.openmrs.org/browse/TRUNK-5031